lighthouse-cli/test/smokehouse/run-smoke.js

```diff
@@ -70,6 +78,7 @@ const SMOKETESTS = [{
   id: 'ttci',
   expectations: 'tricky-ttci/expectations.js',
   config: 'lighthouse-core/config/default-config.js',
+  batch: 'parallel-third',
```

Comment:
this one seems particularly important to run on its own, if it's always really really slow then the expectation it's asserting (that TTI is at least X seconds) becomes meaningless

Reply:
mmmmmmmhmmmmmmm have we seen it flake yet? i kinda dont want to break it into a solo if we don't have to. :)

Reply:
it's not that it will flake, it's that we shouldn't both running it at all if it's going to be running very slowly because the only assertion it makes is that the TTI is at least some value based on tricky properties of the page, but if it always passes because travis so doggone slow then no point in having it